packages/asset-service/src/service/descriptions.json

```diff
@@ -2,11 +2,22 @@
+  "eip155:1/erc20:0xed279fdd11ca84beef15af5d39bb4d4bee23f0ca": "This token represents a liquidity pool on Curve, a leading decentralized exchange. Holders earn fees from users trading in the pool, and can also deposit the LP token on [ShapeShift](/defi/earn) to earn yield. This liquidity pool contains [LUSD](/assets/eip155:1/erc20:0x5f98805a4e8be255a32880fdec7f6728c6568ba0) and [3CRV](/assets/eip155:1/erc20:0x6c3f90f043a72fa612cbac8115ee7e52bde6e490). [3CRV](/assets/eip155:1/erc20:0x6c3f90f043a72fa612cbac8115ee7e52bde6e490) is another curve LP token that consists of [DAI](/assets/eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f), [USDT](/assets/eip155:1/erc20:0xdac17f958d2ee523a2206206994597c13d831ec7), and [USDC](/assets/eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48). To acquire [crvLUSD](/assets/eip155:1/erc20:0xed279fdd11ca84beef15af5d39bb4d4bee23f0ca), deposit [LUSD](/assets/eip155:1/erc20:0x5f98805a4e8be255a32880fdec7f6728c6568ba0, [DAI](/assets/eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f), [USDT](/assets/eip155:1/erc20:0xdac17f958d2ee523a2206206994597c13d831ec7), and/or [USDC](/assets/eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48) on [Curve](https://curve.fi/lusd/deposit). Please be aware that as liquidity pools are composed of differently-priced assets, they are subject to impermanent loss but because all of the assets in the pool are pegged to $1, impermanent loss is unlikely and would only result if one of the assets in the pool lost its peg.",  

```

Comment:
There's a missing closing parenthese resulting in a broken `deposit LUSD` link and missing `DAI` link:
`assets/eip155:1/erc20:0x5f98805a4e8be255a32880fdec7f6728c6568ba0,%20[DAI](/assets/eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f`

Reply:
once this is fixed then it's good to go from a product standpoint! Thanks @gomesalexandre!

Reply:
@gomesalexandre done